docs/src/terminology.md

```diff
@@ -179,6 +189,12 @@ A [program](#program) with the ability to interpret the binary encoding of other
 
 The duration of time for which a [validator](#validator) is unable to [vote](#ledger-vote) on another [fork](#fork).
 
+## message
+
+The structured contents of a [transaction](#transaction). Generally containing a header, array of account addresses, recent [blockhash](#blockhash), and an array of [instructions](#instruction).
+
+Learn more about the [message formatting](./developing/programming-model/transactions.md#message-format) inside of transactions here.
```

Comment:
Is it better to be the following for SEO?

```
Learn more about the [message formatting inside of transactions](./developing/programming-model/transactions.md#message-format).
```

Reply:
I think the SEO effects would be similar, but I like the idea of having the link text to be more specific

Reply:
Link text updated in next push to PR